repository/helper.go

```diff
@@ -0,0 +1,42 @@
+package repository
```

Comment:
Please seperate this file to begin_repository,  rollback_repository,commit_repository